### PR TITLE
add download history element #100

### PR DESCRIPTION
**Pull request by ghUserrrr**

### CODE REVIEW

File: components/GetHistory/GetHistory.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    const currentDate = new Date();
+    const formattedDate = currentDate.getDate() + "." + (currentDate.getMonth() + 1) + "." + currentDate.getFullYear()
+     + "_" + currentDate.getHours() + "." + currentDate.getMinutes();
+
+    const historyDataYaml = YAML.stringify(historyData)
+
+    const a = document.createElement("a")
+    const file = new Blob([historyDataYaml], {type: 'text/plain'});
+    a.href = URL.createObjectURL(file);
+    a.download = `gamerecord_${formattedDate}.txt`;
```
Comment (utgarda) [inline components/GetHistory/GetHistory.tsx:143]: Which game, checkers or ttt? Who played it?
Please make it 
gj_arbiter_0x1234_checkers_0x1234_<game id>_<date>_<0xabcd>_<0x1234>. yaml

**Review by utgarda — CHANGES_REQUESTED** [review]
Please add contract addresses and player addresses to the filename.

File: components/GetHistory/GetHistory.tsx (hunk 1 above)
Comment (ghUserrrr) [reply components/GetHistory/GetHistory.tsx:143]: ok

Comment (ghUserrrr) [reply components/GetHistory/GetHistory.tsx]: changed to "gamerecord_0x1f0b_checkers_731_0xdC5f_0x4114_2023-02-03_15-24.yaml"

**Review by poludnev — APPROVED** [review]

**Review by utgarda — APPROVED** [review]

**Pull request merged**